solver_options:
  regularization: 1e-4

stack: [
  [left_ee, right_ee],
  [postural]
]

constraints: [
  jlim,
  vlim,
  Wheel_FL,
  Wheel_FR, 
  Wheel_HR, 
  Wheel_HL
]

right_ee:
  name: right_ee
  type: Cartesian 
  distal_link: dagana_2_top_link
  base_link: pelvis
  lambda: 0.0

left_ee:
  name: left_ee
  type: Cartesian 
  distal_link: arm1_8
  base_link: pelvis
  lambda: 0.0

Wheel_FL:
    type: Cartesian
    distal_link: wheel_1
    base_link: world
    lambda: 0.1

Wheel_FR:
    type: Cartesian
    distal_link: wheel_2
    base_link: world
    lambda: 0.1

Wheel_HL:
    type: Cartesian
    distal_link: wheel_3
    base_link: world
    lambda: 0.1

Wheel_HR:
    type: Cartesian
    distal_link: wheel_4
    base_link: world
    lambda: 0.1

postural:
    type: Postural
    lambda: 0.01

jlim: 
  type: JointLimits

vlim:
  type: VelocityLimits